### PR TITLE
Enhance EtcdCoordinator if disconnected from etcd server for a long time

### PR DESCRIPTION
**Pull request by kuaikuai**
- Why submit this pull request?

when EtcdCoordinator cannot connect to etcd cluster for a long time, 
the information registered in etcd will expire and refresh() function will not work.

- How to fix?
When refresh() finds that the key does not exist, an exception is thrown, and we catch the exception to rewrite the registration information.

### CODE REVIEW

**Comment by wu-sheng** [comment]
@kezhenxu94 @JaredTan95  @wayilau Could you recheck this? Look like make sense.

File: oap-server/server-cluster-plugin/cluster-etcd-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/etcd/EtcdCoordinator.java [hunk 1]
```diff
@@ -113,7 +114,13 @@ private void renew(EtcdClient client, String key, String json) {
             try {
                 client.refresh(key, KEY_TTL).send().get();
             } catch (Exception e) {
-
+                try {
+                    client.put(key, json).ttl(KEY_TTL)
+                        .setRetryPolicy(new RetryNTimes(1, 10))
```
Comment (wayilau) [inline oap-server/server-cluster-plugin/cluster-etcd-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/etcd/EtcdCoordinator.java]: it looks good to me. Just one point that there is a loop already, I think it needs not another retryPolicy. What about you ?

Comment (kezhenxu94) [inline oap-server/server-cluster-plugin/cluster-etcd-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/etcd/EtcdCoordinator.java]: Retry 10 times every 1 millisecond? do we really need that frequently?

File: oap-server/server-cluster-plugin/cluster-etcd-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/etcd/EtcdCoordinator.java [hunk 2]
```diff
@@ -113,7 +114,13 @@ private void renew(EtcdClient client, String key, String json) {
...
+                        .send()
+                        .get();
+                } catch (Exception ee) {
```
Comment (kezhenxu94) [inline oap-server/server-cluster-plugin/cluster-etcd-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/etcd/EtcdCoordinator.java]: We'd better not swallow the exception, at least, log a warning message to notify users that the Etcd connection is unstable or something

File: oap-server/server-cluster-plugin/cluster-etcd-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/etcd/EtcdCoordinator.java (hunk 1 above)
Comment (kuaikuai) [reply oap-server/server-cluster-plugin/cluster-etcd-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/etcd/EtcdCoordinator.java]: you are right,  I remove retryPolicy.

File: oap-server/server-cluster-plugin/cluster-etcd-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/etcd/EtcdCoordinator.java (hunk 2 above)
Comment (kuaikuai) [reply oap-server/server-cluster-plugin/cluster-etcd-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/etcd/EtcdCoordinator.java]: ok

**Comment by wu-sheng** [comment]
@kuaikuai Welcome to be 171st contributor

**Pull request merged**